modules/apps/site-navigation/site-navigation-taglib/src/main/java/com/liferay/site/navigation/taglib/servlet/taglib/BreadcrumbTag.java

```diff
@@ -58,13 +66,34 @@ protected String getPage() {
 
 	@Override
 	protected void setAttributes(HttpServletRequest httpServletRequest) {
+		List<BreadcrumbEntry> breadcrumbEntries = _breadcrumbEntries;
+
+		try {
+			_serviceTracker = ServiceTrackerFactory.open(
```

Comment:
I had to use this cause
 SF : `Do not use com.liferay.registry classes in modules, see LPS-62989: ./modules/apps/site-navigation/site-navigation-taglib/src/main/java/com/liferay/site/navigation/taglib/servlet/taglib/BreadcrumbTag.java (SourceCheck:JavaModuleIllegalImportsCheck)`
I tried other methods, but this is the simplest I found